### PR TITLE
Update validator-stake.md

### LINKED ISSUE #5737: Document procedure for deactivating stake
During TdS DR 4, there was some confusion about how to deactivate stake.  Totally understandable as we didn't really document how much.

### PR DESCRIPTION
**Pull request by rob-solana**
Fixes #5737

### CODE REVIEW

File: book/src/validator-stake.md [hunk 1]
```diff
@@ -24,15 +24,17 @@ $ solana redeem-vote-credits ~/validator-config/stake-keypair.json ~/validator-v
...
+safely remove a validator from the network, first deactivate its stake.
 
 Stake can be deactivated by running:
 ```bash
 $ solana deactivate-stake ~/validator-config/stake-keypair.json ~/validator-vote-keypair.json
 ```
 
 The stake will cool down, deactivate over time.  While cooling down, your stake will continue to earn
-rewards.
+rewards.  Only after stake cool down is it safe to turn off your validator or withdraw it from the network.
```
Comment (CriesofCarrots) [inline book/src/validator-stake.md]: this "cooldown" s/b one word

Comment (CriesofCarrots) [reply book/src/validator-stake.md]: Also, can you add a sentence about how validators can tell if their stake is cooled down and safe to withdraw?

Comment (rob-solana) [reply book/src/validator-stake.md]: I'd love to, but we don't have an API for that, do we?

Comment (CriesofCarrots) [reply book/src/validator-stake.md]: Ah. No, I'm getting ahead of myself. Nm

Comment (rob-solana) [reply book/src/validator-stake.md]: well, in the case of single stake, single-validator, the live stakes API would show zero stake...

it's important for validators to know what their stake is before bailing and the current API gives them that...

Comment (rob-solana) [reply book/src/validator-stake.md]: as for "safe to withdraw", just keep trying withdraw() until balance is zero ;)

**Pull request merged**